license.txt

```diff
@@ -1,40 +1,8 @@
-WooCommerce - eCommerce for WordPress
-
-Copyright 2015 by the contributors
-
-This program is free software; you can redistribute it and/or modify
-it under the terms of the GNU General Public License as published by
-the Free Software Foundation; either version 3 of the License, or
-(at your option) any later version.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-GNU General Public License for more details.
-
-You should have received a copy of the GNU General Public License
-along with this program; if not, write to the Free Software
-Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
-
-This program incorporates work covered by the following copyright and
-permission notices:
-
-  Jigoshop is Copyright (c) 2011 Jigowatt Ltd.
-  http://jigowatt.com - http://jigoshop.com
-
-  Jigoshop is released under the GPL
-
-and
-
-  WooCommerce - eCommerce for WordPress
-
-  WooCommerce is Copyright (c) 2015 WooThemes
-
-  WooCommerce is released under the GPL
+ClassicCommerce - eCommerce for ClassicPress and is released under the GPL
```

Comment:
The license text should be left in place. Also these changes should be backed out unless they are directly related to fixing the build.